### DISCUSSION TITLE
Why does Atlas-OS require an internet connection?

### DISCUSSION THREAD

**Discussion by sKGu8gxV** [Atlas-OS/Atlas#1260] [Help & Support] [answered]
First off, thanks to all who have helped in the creating of this incredible piece of software!

 I wondere why does Atlas-OS require an internet connection? Everything should be selfcontained in the atlas executable and the playbook?

**Comment by the-P1neapple** [comment, marked as answer]
Hi @sKGu8gxV,
That is not the case. For additional software such as c++ and directx redistrubutables and web browsers. They are downloaded over the internet to ensure you are receiving the latest versions.

**Reply by sKGu8gxV** [reply]
hi the-P1neapple,
Thanks for the reply!
May I ask if it's possible to disable this? I have this already on another disk and would prefer to install this manually, especially browsers.

**Reply by sKGu8gxV** [reply]
Oh and if anyone ever finds this, it is the case. You literally cannot go on in installation of atlasos doesn't find an active internetconnection.
